automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -393,3 +393,234 @@ groups:
       summary: "Missing blocks observed on {{ $labels.testnet }}"
       description: "{{ $value }} Missing block(s) observed on network {{ $labels.testnet }}."
       runbook: "https://www.notion.so/minaprotocol/Archive-Node-Metrics-9edf9c51dd344f1fbf6722082a2e2465"
+
+- name: Berkeley syncStatus related alerts
+  rules:
+  - alert: HighDisconnectedBlocksPerHour
+    expr: max by (testnet) (increase(Coda_Rejected_blocks_no_common_ancestor {testnet="berkeley",syncStatus!="INIT"|"BOOTSTRAP"|"CATCHUP"} [${alert_timeframe}])) > 3
```

Comment:
What's the reason for not using `syncStatus="Synced"` instead of enumerating all the values it shouldn't be?

Reply:
Also, `testnet="berkeley"` can be a variable. Like `$labels.testnet`, you could add a new variable `berkeley_testnet` and set it once.

Reply:
Label not being present is a different issue that probably needs to tackled outside the alert system. Like maybe when verifying deployments or maybe in health checks. And it will be true for any existing or new labels. So, working around that in alert rules is not the solution imo. 
On the other hand making `!=` not only hides what you're looking for, in this case, is it `Synced` or `Offline`? but also adds additional overhead of updating alerts every time there's a new sync status even if it has not impact on these rules.